iceoryx_utils/include/iceoryx_utils/design_pattern/creation.hpp

```diff
@@ -59,19 +60,33 @@ class Creation
         return iox::cxx::success<>();
     }
 
-    bool isInitialized() const noexcept
+    Creation() noexcept = default;
+
+    Creation(Creation&& rhs) noexcept
     {
-        return m_isInitialized;
+        *this = std::move(rhs);
     }
 
-  protected:
-    void moveCreationPatternValues(Creation&& other)
+    Creation& operator=(Creation&& rhs) noexcept
     {
-        m_isInitialized = other.m_isInitialized;
-        m_errorValue = other.m_errorValue;
-        other.m_isInitialized = false;
+        if (this != &rhs)
+        {
+            m_isInitialized = rhs.m_isInitialized;
+            m_errorValue = rhs.m_errorValue;
```

Comment:
Should move the rhs.m_errorValue, in case ErrorType has a move operator=

Reply:
fixed